rust/operator-binary/src/main.rs

```diff
@@ -40,9 +40,6 @@ struct Opts {
 
 #[derive(clap::Parser)]
 struct OpaRun {
-    #[clap(long, env)]
-    opa_bundle_builder_clusterrole: String,
-
```

Comment:
Just pointing out that an operator CLI option has been removed (because it was ultimately unused).